src/ert/config/ensemble_config.py

```diff
@@ -130,26 +124,12 @@ def make_field(field_list: list[str | dict[str, str]]) -> FieldConfig:
 
             # Use field-specific grid if provided,
             # otherwise fall back to global grid.
-            if grid_file_path is not None:
-                try:
-                    dims = get_shape(grid_file_path)
-                except Exception as err:
-                    raise ConfigValidationError.with_context(
-                        f"Could not read grid file {grid_file_path}: {err}",
-                        grid_file_path,
-                    ) from err
-            else:
+            if grid_file_path is None:
                 grid_file_path = global_grid_file_path
-                dims = global_dims
 
-            if dims is None:
-                raise ConfigValidationError.with_context(
-                    f"Grid file {grid_file_path} did not contain dimensions",
-                    grid_file_path,
-                )
             assert grid_file_path is not None
```

Comment:
you can move this assert before line == 128 and just do `assert global_grid_file_path is not None`